lua/harpoon/ui.lua

```diff
@@ -63,13 +67,47 @@ local function get_menu_items()
 
     for _, line in pairs(lines) do
         if not utils.is_white_space(line) then
-            table.insert(indices, line)
+            local item = state.list_by_label[line]
+            if (item ~= nil) then
+                table.insert(indices, item.path)
+            else
+                table.insert(indices, line)
+            end
         end
     end
 
     return indices
 end
 
+-- This method will be moved to the readme as an example once the PR is not a DRAFT
```

Comment:
The readme, or is there any suggested location for an example?

Reply:
The readme would be great.  How are you wanting to integrate this?  I would assume options, but I am unsure if my config makes it all through the program.  this was my first lua plugin for real, and it sucks.

Reply:
Yeah, through the options, we can add an example in the readme, and provide builtin examples so that people do not need to repeat the same code everywhere, and if you think it is a good idea, people could still contribute their builtins (I think this would be similar to null-ls configs)


This is also my first Lua plugin (contribution). Having spent somedays with more readable paths was a good addition, I'm going to complete it and add automated tests later this week